test/tools/router_check/test/route_tests.sh

```diff
@@ -13,5 +13,5 @@ TESTS=("ContentType" "ClusterHeader" "HeaderMatchedRouting" "Redirect" "Redirect
 # Testing expected matches
 for t in "${TESTS[@]}"
 do
-  "${PATH_BIN}" "${PATH_CONFIG}/${t}_router.json" "${PATH_CONFIG}/${t}_expected.json"
+  "${PATH_BIN}" "${PATH_CONFIG}/${t}_router.json" "${PATH_CONFIG}/${t}_golden.json"
```

Comment:
same as above. .json.golden. This way, the files that you generate can be the normal jsons.. e.g., you could compare weighted.json with weighted.json.golden. Once again, its a matter of preference/style. So, please feel free to ignore this, as long as you use a consistent style.